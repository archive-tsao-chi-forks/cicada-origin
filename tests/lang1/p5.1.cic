id : (Nat) -> Nat
id = (x) => x

_ : (Nat) -> (Nat) -> Nat
_ = (_) => (_) => id(id(id(id(100))))

@show _
