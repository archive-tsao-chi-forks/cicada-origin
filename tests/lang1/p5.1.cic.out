(_1) => (_2) => 100: (Nat) -> (Nat) -> Nat
